### ISSUE TITLE
Debugging in nvim

### ISSUE THREAD

**Issue by Jendker** [wookayin/dotfiles#48]
Do you currently have a solution for debugging in nvim or is it something you would do in other editor?

**Comment by wookayin** [comment]
Yes, I have an incomplete yet quite working version of nvim-DAP integration. I have been using it for many months and I'm pretty happy with it (FYI I mostly code in python, lua, and C++). Let me share a preview version or PR soon!

**Comment by Jendker** [comment]
That would be great! Your dotfiles are very clean, would love to use your preview version as an inspiration. Python and C++ would be very interesting to me as well.

**Comment by wookayin** [comment]
See #52.
